core/src/main/scala/com/avast/grpc/jsonbridge/GrpcJsonBridge.scala

```diff
@@ -0,0 +1,11 @@
+package com.avast.grpc.jsonbridge
+
+import io.grpc.BindableService
+
+import scala.concurrent.Future
+
+trait GrpcJsonBridge[Service <: BindableService] {
+  def invokeGrpcMethod(name: String, json: String, headers: Seq[GrpcHeader] = Seq.empty): Option[Future[String]]
```

Comment:
What is the meaning of `Option[Future[String]]`? Please add a comment.

Reply:
ok